oci/spec_unix.go

```diff
@@ -1,5 +1,3 @@
-// +build !windows
```

Comment:
remove the `unix` suffix from the filename

Reply:
I was trying to keep it mostly the same but you are right. Fixed.